README.md

```diff
@@ -742,12 +742,12 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Muhammad Raza](https://github.com/illusiveCode)
   - [Muhammed Fayad](https://github.com/Nyctophilus)
   - [Muiruriscode](https://github.com/Muiruriscode)
+  - [Mujahid Ali](https://github.com/MujahidAli954)
   - [Mukul Sharma](https://github.com/mukul98s)
   - [Mulyadi](https://github.com/Urcane)
   - [Musab Habeeb](https://github.com/Musab1258)
   - [Muthu Annamalai V.](https://github.com/muthuannamalai12)
   - [Maryam Farshbafi](https://github.com/MaryamFarshbafi)
```

Comment:
I think the line below should be added back.
```suggestion
  - [Maryam Farshbafi](https://github.com/MaryamFarshbafi)

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient